src/components/Column/Column.jsx

```diff
@@ -24,6 +28,9 @@ class Column extends Component {
             {this.props.details}
           </div>          
         </div>
+        <div className="column-addjob">
+          {this.addJob()}
```

Comment:
We don't need a function here. In between JSX tags we can just put the content. 
So:
```<div className="column-addjob">
    Add Job
</div>```